elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -102,8 +101,11 @@ def start(arguments={})
             return false
           end
 
+          # Flag to clear the cluster. May be passed directly with args or via ENV variables.
+          clear_cluster = !!arguments[:clear] || (ENV.fetch('TEST_CLUSTER_CLEAR', 'true') != 'false')
```

Comment:
Because it uses a temporary variable used only in the line below. If you prefer I'll move it there.